openedx/core/djangoapps/discussions/models.py

```diff
@@ -130,8 +130,10 @@ class DiscussionsConfiguration(TimeStampedModel):
         db_index=True,
         unique=True,
         max_length=255,
+        # pylint: disable=line-too-long
         # Translators: A key specifying a course, library, program, website, or some other collection of content where learning happens.
```

Comment:
can't you wrap translators comments?

Reply:
I'll try that...

Reply:
yeah: https://github.com/edx/edx-platform/blob/master/common/djangoapps/util/date_utils.py#L141

Reply:
Yeah, I was erring on the side of caution initially because I wasn't sure.
The docs _don't_ say you can't do it, though none of the examples show it:
- https://docs.djangoproject.com/en/3.1/topics/i18n/translation/#comments-for-translators

_But_ we definitely do it already in `edx-platform`, so I think it's good.